tests/integration/Core/Framework/DataAbstractionLayer/fixture/AttributeEntityWithHydrator.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Integration\Core\Framework\DataAbstractionLayer\fixture;
+
+use Shopware\Core\Framework\DataAbstractionLayer\Attribute\Entity;
+use Shopware\Core\Framework\DataAbstractionLayer\Attribute\Field;
+use Shopware\Core\Framework\DataAbstractionLayer\Attribute\FieldType;
+use Shopware\Core\Framework\DataAbstractionLayer\Attribute\PrimaryKey;
+use Shopware\Core\Framework\DataAbstractionLayer\Entity as EntityStruct;
+
+/**
+ * @internal
+ */
+#[Entity('attribute_entity_with_hydrator', hydratorClass: DummyHydrator::class)]
```

Comment:
Currently few tests are failing: https://github.com/shopware/shopware/actions/runs/12765559819/job/35580098527?pr=6068

To fix `EntityDefinitionHasSinceTest` you may add since parameter:
```suggestion
#[Entity('attribute_entity_with_hydrator', since: '6.6.10.0', hydratorClass: DummyHydrator::class)]
```
To fix `ApiAwareTest` you may change `tests/integration/Core/Framework/Api/fixtures/api-aware-fields.json` (use attribute_entity.* fields as example)

Reply:
Hi @h1k3r, I fixed now. Thank you